extensions/Worker.Extensions.Rpc/src/ConfigurationExtensions.cs

```diff
@@ -13,15 +13,28 @@ internal static class ConfigurationExtensions
         /// </summary>
         /// <param name="configuration">The configuration to get the host URI from.</param>
         /// <returns>The URI representing the functions host.</returns>
+        /// <exception cref="UriFormatException">If <paramref name="configuration"/> contain a value for 'functions-uri', but is not a valid URL.</exception>
         /// <exception cref="InvalidOperationException">
         /// If <paramref name="configuration"/> does not contain the required values.
         /// </exception>
         public static Uri GetFunctionsHostGrpcUri(this IConfiguration configuration)
         {
-            string uriString = $"http://{configuration["HOST"]}:{configuration["PORT"]}";
-            if (!Uri.TryCreate(uriString, UriKind.Absolute, out Uri? grpcUri))
+            Uri? grpcUri;
+            var functionsUri = configuration["functions-uri"];
```

Comment:
As mentioned in my other comment, I think we should use command line switch mapping, so `--functions-uri` maps to `FunctionsUri`* in `IConfiguration`.

Additionally, it may be worth considering moving this all under a config section to reduce risk of collisions - but this can be done in a separate PR if you want.

Reply:
I would be in favor of making any configuration adjustments in this PR to avoid changing this too much just in case anyone ends up using it. Moving this to a configuration section makes sense to me.

Reply:
The new itearation is using switch mapping and using a configuration section for the grpc channel information.